ood-setup.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Install new OOD Portal config
+rm -f /etc/ood/config/ood_portal.yml
+cp /vagrant/ood_portal.yml /etc/ood/config/ood_portal.yml
+/opt/ood/ood-portal-generator/sbin/update_ood_portal
+systemctl try-restart httpd24-httpd.service httpd24-htcacheclean.service
```

Comment:
`ood-setup.sh` is run before apache is started, the service is actually started in the `Vagrantfile` but since you used `try-restart` it won't actually do anything, or at least shouldn't.